src/containers/Tenant/QueryEditor/QueryExplain/QueryExplain.js

```diff
@@ -222,22 +227,36 @@ function QueryExplain(props) {
             <React.Fragment>
                 {!isFullscreen && content}
                 {isFullscreen && <Fullscreen>{content}</Fullscreen>}
-                {renderStub()}
             </React.Fragment>
         );
     };
 
+    const renderError = () => {
```

Comment:
Creating a dedicated Error component would suggest that there is a common way of displaying error which is reused between components. But again, this is just a div with text. I'm not sure what to reuse

Also, there should be a fixed shape of an error to pass it into the component, otherwise it will turn into a mess of code branches trying to figure out how to display an arbitrary data. Until we have an error shape, errors better be parsed in-place

Reply:
In pm decided to think about this as a standalone task.